tests/opensource_demo_orangehrmlive/admin/employment_statuses/test_delete_employment_statuses.py

```diff
@@ -139,7 +141,7 @@ def test_BYT_T190_Eliminar_estado_con_id_decimal(statuses_url, header):
     Prioridad: Media
     """
     payload = json.dumps({"ids": [1.2]})
-    response = requests.delete(statuses_url, headers=header, data=payload)
+    response = OrangeRequest.delete(statuses_url, headers=header, payload=payload)
     assert response.status_code == 404
```

Comment:
En todos tus tests cases falta reutilizar el assert el status code
```suggestion
    assert_status_code(response, expected_status=404)
```